__tests__/system/Map.test.js

```diff
@@ -16,10 +16,8 @@ const setUpBottom = (props={}) => {
 
 describe("Map Component", () => {
    let component;
-   let component2;
```

Comment:
Just to make sure, why did you remove this in the test though?

Reply:
Basically it was not used in the test, I double checked that the test still passes after removing it.

Reply:
Makes sense!